burstphoto/merge/spatial.swift

```diff
@@ -15,7 +15,7 @@ let compute_merge_weight_state = try! device.makeComputePipelineState(function:
 func align_merge_spatial_domain(progress: ProcessingProgress, ref_idx: Int, mosaic_pattern_width: Int, search_distance: Int, tile_size: Int, noise_reduction: Double, uniform_exposure: Bool, exposure_bias: [Int], black_level: [[Int]], color_factors: [[Double]], textures: [MTLTexture], hotpixel_weight_texture: MTLTexture, final_texture: MTLTexture) throws {
     print("Merging in the spatial domain...")
     
-    let kernel_size = Int(16) // kernel size of binomial filtering used for blurring the image
+    let kernel_size = mosaic_pattern_width == 6 ? 8 : 16 // kernel size of binomial filtering used for blurring the image
```

Comment:
Does it make sense to use integer here?     Int(mosaic_pattern_width == 6 ? 8 : 16)

Reply:
Swift infers 8 and 16 to be integers because there's no decimals. The `Int(16)` from before was also not required, it was equivalent to `16`.